mojave/effects/spawners/lootdrop/gunloot.dm

```diff
@@ -375,6 +375,11 @@
 				/obj/item/ammo_box/magazine/ms13/m45,
 				/obj/item/ammo_box/magazine/ms13/m45
 				)
+	var/loot6 = list(
+				/obj/item/gun/energy/ms13/laser/pistol/wattz/magneto,
+				/obj/item/stock_parts/cell/ms13/ec,
```

Comment:
This should really only have one spare EC. Laser weapons get a lot of ammo per energy cell so they only spawn with one as a policy. Same with similar high ammo count weapons like the service rifle, SMGs, etc.